src/core/strvector.c

```diff
@@ -21,13 +21,15 @@
 
 */
 
+#include <string.h>         /* memcpy & co. */
+#include <stdlib.h>
```

Comment:
Normally we order headers from most specific to least specific, meaning that standard headers come last. This helps avoid introducing a bug where an igraph header only works if a standard header is included before it. This is the usual recommendation for C and C++.